searx/plugins/self_info.py

```diff
@@ -25,20 +29,42 @@
 p = re.compile('.*user[ -]agent.*', re.IGNORECASE)
 
 
+# Returns a string with all the information retrieved from ip-api.com's API
+def get_ip(request):
+    x_forwarded_for = request.headers.getlist("X-Forwarded-For")
+
+    if x_forwarded_for:
+        ip = x_forwarded_for[0]
+    else:
+        ip = request.remote_addr
+
+    if ip == '127.0.0.1':
+        ip = ''
+
+    # Find the outgoing proxies settings from the user configuration.
+    outgoing_proxies = settings['outgoing'].get('proxies', None)
+
+    # Initiate a GET request and set the outgoing proxies, if any were set in
+    # settings.yml.
+    ip_info = get('http://ip-api.com/json/' + ip, proxies=outgoing_proxies).json()
```

Comment:
Ok, I can change it to ipinfo.io for now, and look into implementing it internally. The Maxmind datafiles have plenty of information, the same that ipinfo.io provides, so it should be fine.